tests/stub/authorization/test_authorization.py

```diff
@@ -914,9 +929,7 @@ def allocate_connections(n):
         self.assertEqual(hangup_count_pre + 1, hangup_count_post)
 
 
-class TestAuthenticationSchemes(AuthorizationBase):
```

Comment:
Just to note that this updates test names and backend skips might need updating.